src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpFormatting.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Input;
+using Roslyn.Test.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.Extensions.Editor;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpFormatting : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        public CSharpFormatting(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(CSharpFormatting))
+        {
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public void AlignOpenBraceWithMethodDeclaration()
+        {
+            SetUpEditor(@"
+$$class C
+{
+    void Main()
+     {
+    }
+}");
+
+            Editor.FormatDocument();
+            this.VerifyTextContains(@"
+class C
+{
+    void Main()
+    {
+    }
+}");
+        }
+
```

Comment:
AlignCloseBraceWithOpenBrace is covered by http://source.roslyn.io/#Roslyn.Services.CSharp.UnitTests/Formatting/FormattingTests.cs,a7315a6f01a24fe9
OpenBraceOnNewLine is covered by http://source.roslyn.io/#Roslyn.Services.CSharp.UnitTests/Formatting/FormattingTests.cs,2c74d799774b3fb1